src/t8_forest/t8_forest_ghost.h

```diff
@@ -69,6 +69,30 @@ t8_forest_ghost_get_tree_element_offset (t8_forest_t forest, t8_locidx_t lghost_
 t8_locidx_t
 t8_forest_ghost_tree_num_elements (t8_forest_t forest, t8_locidx_t lghost_tree);
 
+/** Retrieves the ghost index of an element in a specific ghost tree.
+ *
+ * \param [in] forest The forest object.
+ * \param [in] lghost_tree The local index of the ghost tree.
+ * \param [in] linear_id The local index of the element.
+ * \param [in] element_level The level of the element.
+ * \param [out] loc_ghost_id The local id of the ghost. -1 if no ghost was found.
```

Comment:
We have to be very careful with the documentation and usage here.
The value of loc_ghost_id is the local **index** of the Ghost within its tree. (The return value of t8_ghost_get_ghost_id_in_tree)
It is not the id as in "The index in the elements + ghosts array that we use for example in ghost_exchange_data".

Reply:
The current documention is in mho too unclear about that.